config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   get "/pages/:page", to: "pages#show"
 
+  resources :schools
```

Comment:
Since the controller only supports show, new and create, for security reasons I would whitelist the supported actions here.